Lesson13/Kadir/conditionalStatements.js

```diff
@@ -89,14 +172,101 @@ election2(14);
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
 
+let height=parseInt(prompt("Input your height(m)= "));
+let weight=parseInt(prompt("Input your weight(kg)= "));
+
+function indexBMI(h, w){
+  if (w/(h^2)<=18.5){
```

Comment:
You have to calculate bmi a few times in this function. Do that as the first step, then use the result of the calculation everywhere.
Also, it would be good to check that the input values that the function receives are as expected (e.g. not negative numbers, strings, etc.). This also applies to other exercises. 
```suggestion
  const bmi = w/(h^2)<=18.5;
  if (bmi){
```